src/fe-gtk/setup.c

```diff
@@ -614,9 +614,7 @@ static const char *const proxytypes[] =
 	N_("SOCKS4"),
 	N_("SOCKS5"),
 	N_("HTTP"),
-#ifdef USE_LIBPROXY
 	N_("Auto"),
```

Comment:
Note this setting will be broken on Windows if (a) you don't build libproxy, or (b) you don't build glib-networking with libproxy support enabled.